README.md

```diff
@@ -182,12 +184,15 @@ Arguments:
   srcPath              path to a directory or file (default: "")
 
 Options:
-  --graph              fixing all file(s) within the graph, which might include files
-                       outside of the current directory
-  --ignore [globs...]  comma-delimited list of globs to ignore eg. '--ignore
-                       tests/**/*,types/**/*' (default: [])
-  -d, --dryRun         do nothing; only show what would happen (default: false)
-  -h, --help           display help for command
+  --graph                   fixing all file(s) within the graph, which might include files
+                            outside of the current directory
+  --ignore [globs...]       comma-delimited list of globs to ignore eg. '--ignore
+                            tests/**/*,types/**/*' (default: [])
+  -d, --dryRun              do nothing; only show what would happen (default: false)
+  -h, --help                display help for command'
+  --skipChecks [checks...]  skips check for packages, useful if a codebase has a novel
```

Comment:
once the change in the command itself has been made just update this to match

Reply:
Done